extensions/amp-pan-zoom/0.1/amp-pan-zoom.js

```diff
@@ -414,23 +414,23 @@ export class AmpPanZoom extends AMP.BaseElement {
       }
     });
 
-    // Override all taps to enable tap events on content
-    this.gestures_.onGesture(TapRecognizer, e => {
-      const event = createCustomEvent(this.win, 'click', null, {bubbles: true});
-      e.data.target.dispatchEvent(event);
-    });
-
     if (!this.disableDoubleTap_) {
       this.gestures_.onGesture(DoubletapRecognizer, e => {
         const {clientX, clientY} = e.data;
-        const newScale = this.scale_ == 1 ? this.maxScale_ : this.minScale_;
-        const dx = (this.elementBox_.width / 2) + this.getOffsetX_(clientX);
-        const dy = (this.elementBox_.height / 2) + this.getOffsetY_(clientY);
-        this.onZoom_(newScale, dx, dy, /*animate*/ true)
+        this.onDoubletapZoom_(clientX, clientY)
             .then(() => this.onZoomRelease_());
       });
+      // Override all taps to enable tap events on content
+      this.gestures_.onGesture(TapRecognizer, e => {
+        const event = createCustomEvent(
```

Comment:
Because it throws: `TypeError: e.data.target.click is not a function`. The `EventTarget` that you get doesn't seem to have the same API as `Element` and somewhere in between the `click` function got stripped?

Reply:
It looks like `click()` exists on `HTMLElement` but not on `Element` (which means you can't `click()` anything that is SVG).